src/savegame_json.cpp

```diff
@@ -454,12 +454,16 @@ void player::load(JsonObject &data)
 
     // Add the earplugs.
     if( has_bionic( "bio_ears" ) && !has_bionic( "bio_earplugs" ) ) {
-        add_bionic("bio_earplugs");
+        // temporary placeholder:
+        std::map<body_part, int> tmp = { num_bp, 1 };
+        add_bionic("bio_earplugs", tmp );
```

Comment:
It might be possible to define the map in line
`add_bionic( "bio_earplugs", { num_bp, 1 } )`

Also style (initial spacing before string argument)